airflow-core/docs/authoring-and-scheduling/connections.rst

```diff
@@ -47,5 +47,5 @@ Airflow allows to define custom connection types. This is what is described in d
 :doc:`apache-airflow-providers:index` - providers give you the capability of defining your own connections.
 The connection customization can be done by any provider, but also
 many of the providers managed by the community define custom connection types.
-The full list of all providers delivered by ``Apache Airflow community managed providers`` can be found in
+The full list of all connections delivered by ``Apache Airflow community managed providers`` can be found in
```

Comment:
I noticed that typo - I can separate it if needed - but not worth IMHO

Reply:
Let's keep it, it is fine 👌